requirements/docs.txt

```diff
@@ -8,6 +8,7 @@ docutils >= 0.18
 ipykernel
 ipython
 jinja2 != 3.1
+lxml
```

Comment:
This won't actually fix your RTD build. According to your `.readthedocs.yml` file, you're installing your doc dependencies using `pip install .[docs]`. Therefore, you should add this dependency to the `docs` extra in setup.cfg: https://github.com/HinodeXRT/xrtpy/blob/691cf6c9b20463bc5f62dd76250059e2759b5749/setup.cfg#L60. 

Also, I would just install `sunpy[net]`. This will give you all the dependencies you need to run Fido.

Reply:
Thanks! Regarding installing just sunpy.net then you mean in setup.cfg we should replace 
`sunpy >= 3.0.0`
with
`sunpy[net] >= 3.0.0`
?

Reply:
No, I think you'll want to leave the `sunpy` dependency in `setup.cfg` as is, but you'll want to add another entry under the `docs` extra requirements. This is because you only need this functionality for building the docs, but not for actually using the `xrtpy` package.

My suggested change below clarifies where this change should go more explicitly.